### ISSUE TITLE
[Audio] Audio doesn't gracefully errors on failed startup

### ISSUE THREAD

**Issue by Drapersniper** [Cog-Creators/Red-DiscordBot#3389]
```
[2020-01-16 23:02:43] [ERROR] red: AttributeError
Traceback (most recent call last):
  File "c:\windows\system32\redenv\lib\site-packages\discord\ext\commands\core.py", line 79, in wrapped
    ret = await coro(*args, **kwargs)
  File "c:\windows\system32\redenv\lib\site-packages\redbot\cogs\audio\audio.py", line 2738, in play
    elif not await is_allowed(ctx.guild, f"{query}", query_obj=query):
  File "c:\windows\system32\redenv\lib\site-packages\redbot\cogs\audio\utils.py", line 90, in is_allowed
    global_whitelist = set(await _config.url_keyword_whitelist())
AttributeError: 'NoneType' object has no attribute 'url_keyword_whitelist'
```

**Comment by Drapersniper** [comment]
Blocked by #3459

**Comment by Drapersniper** [comment]
Fixed by #3459

**Comment by Drapersniper** [comment]
This should already be fixed in the Feature branch, leaving this note here for whoever is going to continue working on it>

The reason here is due to the hotreload issues and Audio having globals that were badly initialized during a hot-reload
